### PR TITLE
JA: Add ja/lite/microcontrollers/library.md

### PR DESCRIPTION
**Pull request by zinziroge**
Original Document:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/microcontrollers/library.md

### CODE REVIEW

File: site/ja/lite/microcontrollers/library.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+たとえば、`kernels/cmsis-nn` はいくつかの最適化されたカーネルを含み、Arm のCMSIS-NN ライブラリを使用します。
+
+最適化されたカーネルを使ったプロジェクトを生成するためには、以下のコマンドを使用します。
+`<subdirectory_name>` は最適化を含むサブディレクトリの名前に置き換えます。
+
+```bash
+make -f tensorflow/lite/micro/tools/make/Makefile TAGS=<subdirectory_name> generate_projects
+```
+
+新しいサブフォルダーを作成することで、あなた自身による最適化を追加することができます。我々は、新しい最適化実装へのプルリクエスト奨励しています。
```
Comment (AseiSugiyama) [inline site/ja/lite/microcontrollers/library.md:136]: [weak suggestion]

```suggestion
新しいサブフォルダーを作成することで、あなた自身による最適化を追加することができます。我々は、最適化された新しい実装について、プルリクエストを奨励しています。
```

File: site/ja/lite/microcontrollers/library.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
+# C++ライブラリを理解する
+
+マイクロコントローラ向け TensorFlow Lite の C++ライブラリ は
+[TensorFlow repository](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/experimental/micro)
+の一部です。
+読みやすく、修正しやすく、よく検証されており、統合しやすく、また、標準のTensorFlow Lite と互換性があるように設計されています。
```
[the remark this answers is not in the document]
Comment (zinziroge) [reply site/ja/lite/microcontrollers/library.md:6]: I agreed on how to use the word "valid".
I used your suggestion. Thank you for your concern.

File: site/ja/lite/microcontrollers/library.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+以下の資料は、C++ライブラリの基本構造の概要を示しており、プロジェクト作成に関する情報を提供します。
+
+## ファイル構造
+
+[`micro`](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/experimental/micro)
+のルートディレクトリは、比較的簡単な構造です。
+しかし、広大な TensorFlow レポジトリの中に配置されているので、われわれはスクリプトと生成済みプロジェクトファイルを作成しました。そのプロジェクトファイルは、さまざまな組込み開発環境の内に隔離して、関連のあるソースファイルを提供します。
```
[the remark this answers is not in the document]
Comment (zinziroge) [reply site/ja/lite/microcontrollers/library.md:14]: I used your suggestion.

File: site/ja/lite/microcontrollers/library.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+### 重要なファイル
+
+マイクロコントローラ向け TensorFlow Lite のインタープリタを使うためにもっとも重要なファイルは、プロジェクトのルートに配置されており、テストが付属しています。
+
+-   [`all_ops_resolver.h`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/micro/kernels/all_ops_resolver.h)
+    または
+    [`micro_mutable_op_resolver.h`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/micro/micro_mutable_op_resolver.h)
+    は、モデルを実行するインタープリタによって使用される演算を提供します。
+    `all_ops_resolver.h` はすべての利用可能な演算を含むので、多くのメモリを使用します。製品アプリケーションにおいては、`micro_mutable_op_resolver.h` を使用するべきであり、モデルが必要とする演算のみを読み込みます。
```
[the remark this answers is not in the document]
Comment (zinziroge) [reply site/ja/lite/microcontrollers/library.md:24]: I used your suggestion.

File: site/ja/lite/microcontrollers/library.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+    オペレーションの実装と関連するコードを含みます。
+-   [`tools`](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/micro/tools)、
+    ビルドツールとその出力を含みます。
+-   [`examples`](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/micro/examples)、
+    サンプルコードを含みます。
+
+## 新規プロジェクトを始める
+
+*Hello World* サンプルを新規プロジェクトのテンプレートととして使うことを推奨しています。
+この節の以下の指示に従うことで、選択したプラットフォームのサンプルを得られます。
```
[the remark this answers is not in the document]
Comment (zinziroge) [reply site/ja/lite/microcontrollers/library.md:47]: I used your suggestion.

File: site/ja/lite/microcontrollers/library.md (hunk 1 above)
Comment (zinziroge) [reply site/ja/lite/microcontrollers/library.md:136]: I used your suggestion.

**Comment by AseiSugiyama** [comment]
@zinziroge Great work! LGTM 👍

**Pull request merged**